infra/gcb/build_project.py

```diff
@@ -20,7 +20,7 @@
 from oauth2client.service_account import ServiceAccountCredentials
 from googleapiclient.discovery import build
 
-BUILD_TIMEOUT = 10 * 60 * 60
+BUILD_TIMEOUT = 20 * 60 * 60
```

Comment:
maybe even 30 minutes is ok ?

Reply:
The build takes around 2 hours on local docker container :(

Reply:
These are hours :)

Reply:
Yeah, but we build several configurations, and the cloud builder is likely a bit less powerful than our desktops :)

Just give me a second to double check something in the logs. I think I won't increase this timeout limit, actually.

Reply:
Yeah, remove this. Let's see if we can build a single configuration and how long it takes. Then we'll think what we should do next.